rosidl_generator_py/resource/_msg_support.c.em

```diff
@@ -156,8 +156,49 @@ PyObject * @('__'.join(type_.namespaces + [convert_camel_case_to_lower_case_unde
 @[  end if]@
 @[end for]@
 
+@# TODO(aamaslov) duplicated struct definition here
```

Comment:
Can we remove the duplicate struct definition and use only one?

Reply:
Struct is moved to `@(package_name)_decl.h` header file.